documentation/modules/con-jmxtrans.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
```

Comment:
This concept doesn't appear to be included anywhere. I don't think a separate concept for JmxTrans is actually necessary. Why not include this content in `assembly-jmxtrans.adoc`?

Reply:
Have done so now :)